### ISSUE TITLE
Kapua with docker seems broken in 1.0.0

### ISSUE THREAD

**Issue by luigisaetta** [eclipse/kapua#2151]
**Describe the bug**
I have tried to test the 1.0.0 Kapua docker deployments and I'm not able to create
users in any way

I tried with latest: results, not able to set  account settings, after creating an account.

Then switched back to 1.0.0.-M7. Able to create an account and raise limits (set to unlimited)
but when I try to create a user I get serialization error.

 Type 'com.extjs.gxt.ui.client.data.BasePagingLoadConfig' was not assignable to 'com.google.gwt.user.client.rpc.IsSerializable' and did not have a custom field serializer. For security purposes, this type will not be deserialized.

**To Reproduce**

1. clone git repository
2. set the version want to test (latest and then M7
3. ./docker-deploy.sh
4. enter in Kapua UI
5. Create a first account
6. Set to every boolean setting has to be set on **true**
7. Try to create an user for the account

**Expected behavior**

**Screenshots**

**Version of Kapua**
1.0.0 and 1.0.0-M7

**Type of deployment**
[ ] Local Vagrant deployment
[X ] Docker
[ ] Openshift (in its variants) 
[ ] Others

**Main component affected**
[ ] Console (in case of console please report info on which browser you encountered the problem)

**Additional context**
Basically, not able to do anything with kapua

**Comment by lorthirk** [comment]
Hello,

Those two look like two different issues. So, I'm going to try to support you for both, but it may be a good idea to split them in separate issues if we can't find a solution quickly for at least one of them.

For the first one, I would need some more details on the error you get when you try to set the account. For the second one, instead, I'd like to know if you're building the images from sources in your local repo and, if yes, the exact Maven build command.

**Comment by luigisaetta** [comment]
Hello,

Ok, I have made some progress and understood what happened. 
In summary, It works and you can close this issue.

I mentioned M7 because I was not able to follow step needed to setup an account with "latest".
I think the reason was "not enough memory" (I had only 6GB in my Cloud VM).
Now, to avoid troubles, I have raised to 4 CPU 16 GB.

I'm able to create an account and I'm able to create a user. The only thing is that the user creation took 5 minutes.

But that's another different thing. So, you can close the isse and thanks for your help.

Continuing to test with Kapua 1.0.0

**Comment by lorthirk** [comment]
Great to hear! So, I'm going to close this issue. As for any other feedback, we would love to hear about them; our mailing list kapua-dev@eclipse.org would also be a great place to discuss about them.
